### DISCUSSION TITLE
title should support variables like $__interval

### DISCUSSION THREAD

**Discussion by jelmd** [grafana/grafana#38098] [Dashboards, Panels & Visualizations]
To understand charts better, it is usually essential to know basic parameters like `$__interval`. Unfortunately there seemr rs to be no way to supplement a chart with such a simple information. At least `title` should support such simple variables.

It would be nice, if limits could also use variables or even could support query results. E.g. one uses different GPU cards, which have different warning for temperatures and power usage. Those parameters  gets exposed via metrics (constant value), but cannot be used e.g. for limit or min/max configuration of the chart. That's really bad and inefficient. ...

**Comment by papagian** [comment]
We need some additional information for evaluating this request.

- Would you like to use the builtin `$__interval` variable in panel title or dashboard title?

- Also, about the limits, could you provide a more precise use case? Which query/datasource are you using and what specifically you cannot achieve?

**Comment by jelmd** [comment]
@papagian

Yes, panel title is sufficient. For dashboard title is might be useful if the related dashboard is shown/open, otherwise probably misleading.

E.g. dcgm-exporter exposes:
```
DCGM_FI_DEV_GPU_TEMP,    gauge, GPU temperature (in C).
DCGM_FI_DEV_SLOWDOWN_TEMP, gauge, Slowdown temperature for the device.
DCGM_FI_DEV_SHUTDOWN_TEMP, gauge, Shutdown temperature for the device.
```

`DCGM_FI_DEV_{SLOWDOWN,SHUTDOWN}_TEMP` can be considered as constants, as long as the graphics card doesn't get changed. However the value is different, depending on the GPU model (e.g. for a `RTX 2080 Ti` they are 91 and 94°C, for a V100-SXM2-32GB they are 87 and 90°C,...).

So if I choose a `Gauge` or `Bar gauge` viz, it would be nice, if one is able to set thresholds related to DCGM_FI_DEV_SLOWDOWN_TEMP, e.g. last($query(DCGM_FI_DEV_SLOWDOWN_TEMP)) etc. and if one uses `Graph` viz, to compare `T1` etc. to the same.

There are other similar use cases, e.g. wrt. to PSU: power draw, voltages, ...

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
